### DISCUSSION TITLE
Network request error. Received status 404: error while looking up block for round '22778227': block not found

### DISCUSSION THREAD

**Discussion by Fraccas** [reach-sh/reach-lang#1332] [Q&A] [answered]
Our team is prepping for a MainNet launch and are now moving away from TestNet. 

On TestNet we never saw any error messages when creating smart contracts, but on MainNet we're getting this. 

The contracts are being created and I can purchase from them successfully. Any thoughts on this?

```
retryLoop {
  lab: [ 'Q2ZX: sendrecv m1 undefined', 'getTimeSecs' ],
  retries: 0,
  e: Error: Network request error. Received status 404: error while looking up block for round '22778227': block not found
      at Request.callback (@reach-sh\stdlib\node_modules\superagent\lib\node\index.js:883:15) 
      at @reach-sh\stdlib\node_modules\superagent\lib\node\index.js:1127:20
      at IncomingMessage.<anonymous> (@reach-sh\stdlib\node_modules\superagent\lib\node\parsers\image.js:10:5)
      at Stream.emit (events.js:315:20)
      at Unzip.<anonymous> (@reach-sh\stdlib\node_modules\superagent\lib\node\unzip.js:53:12) 
      at Unzip.emit (events.js:315:20)
      at endReadableNT (internal/streams/readable.js:1327:12)
      at processTicksAndRejections (internal/process/task_queues.js:80:21) {
    status: 404,

**Comment by jeapostrophe** [comment, marked as answer]
Is your program actually breaking? It sounds like it isn't. This appears to me a "normal operation" message. We are querying for the next block and the way nodes work is that they give a 404 when it isn't there. That's why the error is ignored. It's not big deal that you see it in the log. (Again, maybe I'm wrong and this IS crashing your program?)

**Reply by Fraccas** [reply]
So it seemed to always be successful during the smart contract creation process, though occasionally it did take some time. 

We've been running this on TestNet for about six months now with no issues, not sure what changes when we move to MainNet. We'll continue to keep an eye on this. 

Note: we are on Reach 0.1.10-rc.1

**Reply by Fraccas** [reply]
After a few days of testing, this seems to actually not break the program. 

Thank you for the information.

**Reply by jeapostrophe** [reply]
Excellent!
